ignite/contrib/handlers/throughput_benchmark_wrapper.py

```diff
@@ -0,0 +1,244 @@
+import torch
```

Comment:
Could you please rearrange the imports as 
```
from typing import Callable
import contextlib

import torch

from ignite.engine import Events, Engine
```

Reply:
Done